medium-tests/src/test/java/mediumtest/fixtures/ServerFixture.java

```diff
@@ -414,15 +414,25 @@ private void registerSearchIssueApiResponses() {
 
         var allIssues = issuesPerFilePath.values().stream().flatMap(Collection::stream).collect(toList());
 
-        allIssues.forEach(issue -> mockServer.stubFor(get("/api/issues/search.protobuf?issues=".concat(urlEncode(issue.getKey())).concat("&ps=1&p=1"))
+        allIssues.forEach(issue -> {
+          var searchUrl = "/api/issues/search.protobuf?issues=".concat(urlEncode(issue.getKey()))
+            .concat("&componentKeys=").concat(projectKey)
+            .concat("&ps=1&p=1");
+          if (issue.getKey().contains("PR")) {
```

Comment:
I wanted to somehow add PR param to the request, but only for some issues

Reply:
I think you can use this shortcut to go fast. A more future-proof change could be to set the pull request details on the branch, or to introduce a `withPullRequest`.